qiskit_ibm_runtime/options/layer_noise_learning_options.py

```diff
@@ -0,0 +1,75 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Options for learning layer noise."""
+
+from typing import Union, List
+
+from pydantic import ValidationInfo, field_validator
+
+from .utils import (
+    Unset,
+    UnsetType,
+    primitive_dataclass,
+    make_constraint_validator,
+    skip_unset_validation,
+)
+
+
+@primitive_dataclass
+class LayerNoiseLearningOptions:
+    """Options for learning layer noise.
+
+    .. note::
+        These options are only used when the resilience level or options specify a
+        technique that requires layer noise learning.
+
+    Args:
+        max_layers_to_learn: The max number of unique layers to learn.
+            A ``None`` value indicates that there is no limit.
+            If there are more unique layers present, then some layers will not be learned or
+            mitigated. The learned layers are prioritized based on the number of times they
+            occur in a set of run estimator PUBs, and for equally occurring layers are
+            further sorted by the number of two-qubit gates in the layer. Default: 20.
```

Comment:
The default of 20 is way to high, this could easily timeout an 8 hour job just with learning. In V1 the default was 3 or 4 and we should use the same here unless a user explicitly changes it.

Reply:
Maybe just remove the default for now then and we can add something back in later
```suggestion
            further sorted by the number of two-qubit gates in the layer.
```